File: challenges/codeeval/easy/040/Skhorn.pl
#!/usr/bin/perl

#use strict;
use warnings;

#Sample code to read in test cases:

open(INFILE, "<", $ARGV[0]) or die("Cannot open file $ARGV[0] for reading: $!");
while(my $line = <INFILE>) {
	my @arr_line;
    next if($line =~ m/^\s$/);  # skip blank lines
    $line =~ s/(^\s|\s*$)//g;   # remove leading and trailing spaces
    
    @arr_line = split("",$line);

    for($i = 0; $i < length($line); $i++)
    {
    	 
    	 $num = grep { $_ eq "$i" } @arr_line;

    	 if($num == $arr_line[$i])
    	 {
    	 	push @desc_numb, 1;
    	 }
    	 else
    	 {
    	 	push @desc_numb, 0;
    	 }
    	 	
    }
    $num = grep { $_ eq "1" } @desc_numb;
    if($num == scalar @desc_numb)
    {
    	print "1";
    }
    else
    {
    	print "0";
    }
    print "\n";
    @desc_numb = ();
    
}
close(INFILE);

File: challenges/codeeval/easy/021/Skhorn.py
#!/usr/bin/python

import sys

class sum_digits:

	#file = open('test', 'r')
	sum_dig = 0
	with open(sys.argv[1], 'r') as test_cases:

		for line in test_cases:
			
			for i in range(len(line)):

				if line[i] == '\n':
					break
				sum_dig += int(line[i])

			print sum_dig
			sum_dig = 0



x = sum_digits()